Formula/timew.rb

```diff
@@ -0,0 +1,18 @@
+class Timew < Formula
```

Comment:
Formula name should be `timewarrior` (i.e., source file should be `timewarrior.rb`), and class name should be `Timewarrior`.